apps/platform/components/members/Table.tsx

```diff
@@ -18,7 +19,7 @@ interface TableProps {
   tab: Tab;
   setTab: (tab: Tab) => void;
   members: UserType[];
-  userAccessInCurrentProject: Access;
+  roleInCurrentProject: UserRole;
```

Comment:
```suggestion
  currentRole: UserRole;
```

Lets follow common pattern, eg: `currentProject` , `currentUser`, `currentRole` and so on.

Reply:
Makes sense. I will do that right away